frontend/src/app/settings/modelsources/git-settings/git-settings.component.ts

```diff
@@ -18,60 +28,92 @@ import { DeleteGitSettingsDialogComponent } from 'src/app/settings/modelsources/
   templateUrl: './git-settings.component.html',
   styleUrls: ['./git-settings.component.css'],
 })
-export class GitSettingsComponent implements OnInit {
-  public instances: GitSettings[];
-  gitSettingsForm = new FormGroup({
+export class GitSettingsComponent implements OnInit, OnDestroy {
+  public cmpGitSettings: GitSetting[];
```

Comment:
What does `cmp` stand for in this context?

Reply:
It stands for component, but I also renamed it because I wasn't happy with the name myself